### PR TITLE
fix: chip badge center

### PR DESCRIPTION
**Pull request by edsonsnunes**
Captura de tela de 2022-11-23 14-01-22

badge center

### CODE REVIEW

File: projects/ion/src/lib/chip/chip.component.html [hunk 1]
```diff
@@ -33,6 +33,7 @@
   </div>
 
   <ion-badge
+  class="ionBadge"
```
Comment (iurynogueira) [inline projects/ion/src/lib/chip/chip.component.html:36]:
```suggestion
    class="ionBadge"
```

**Review by iurynogueira — APPROVED** [review]
Only indent. 😄

File: projects/ion/src/lib/badge/badge.component.scss [hunk 2]
```diff
@@ -1,12 +1,16 @@
 @import '../../styles/index.scss';
 
+.ion-badge {
+  display: flex;
+}
+
 span {
```
Comment (mizek1) [inline projects/ion/src/lib/badge/badge.component.scss:7]: This can be put inside `.ion-badge`

**Review by mizek1 — APPROVED** [review]
Great solution!

**Pull request merged**